decidim-core/config/locales/eu.yml

```diff
@@ -2,4 +2,4 @@ eu:
   locales:
     ca: Català
     en: English
-    es: Castellano
\ No newline at end of file
+    es: Castellano
```

Comment:
It's actually very easy. We create a `locale_name` helper:

```ruby
def locale_name(locale)
  I18n.with_locale(locale){ I18n.t("locale_name") }
end
```

Then we just:

```yaml
ca:
  locale_name: Català
```


```yaml
es:
  locale_name: Castellano
```


```yaml
eu:
  locale_name: Euskera
```

```yaml
en:
  locale_name: English
```

💥 

Do you think you could add in in this PR?